src/sentry/ingest/transaction_clusterer/rules.py

```diff
@@ -188,23 +190,33 @@ def get_sorted_rules(project: Project) -> List[Tuple[ReplacementRule, int]]:
     return ProjectOptionRuleStore().read_sorted(project)
 
 
-def update_rules(project: Project, new_rules: Sequence[ReplacementRule]) -> None:
+def update_rules(project: Project, new_rules: Sequence[ReplacementRule]) -> int:
+    """Write newly discovered rules to projection option and redis, and update last_used.
```

Comment:
```suggestion
    """Writes newly discovered rules to project option and Redis, and updates last_used.
```

Reply:
Python has different conventions than Rust:

> The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.

https://peps.python.org/pep-0257/#one-line-docstrings

I'll fix the "Returns" below.